xml/System.Windows.Forms/TextFormatFlags.xml

```diff
@@ -183,7 +183,7 @@
       </ReturnValue>
       <MemberValue>1048576</MemberValue>
       <Docs>
-        <summary>A legacy value that has no effect. It ignored the ampersand (&amp;) prefix character in the text, so that the letter that followed was not underlined, but other mnemonic-prefix characters were still processed.</summary>
+        <summary>It ignores the ampersand (&amp;) prefix character in the text, so that the letter that follows will not be underlined, but other mnemonic-prefix characters are still processed.</summary>
```

Comment:
```suggestion
        <summary>Ignores the ampersand (&amp;) prefix character in the text, so that the letter that follows won't be underlined, but other mnemonic-prefix characters are still processed.</summary>
```